src/Transformer.php

```diff
@@ -39,5 +45,15 @@ public function prepareTransformation()
         return $merged->parseNested();
     }
 
+    public function getOptions(): array
```

Comment:
Why do you need to access publicly to the options here ?

Reply:
I would rename it just `options()`, like the previous`key()`.

Reply:
> Why do you need to access publicly to the options here ?

In fact there is no particular reasons as options will be not accessible outside of the class.

Reply:
Un less we set the property `private`.